dapp/src/components/wizard/pages/ChooseNetwork.jsx

```diff
@@ -1,17 +1,36 @@
 import React, { useState, useEffect } from 'react'
 import { Field, connect as formikConnect, getIn } from 'formik'
-import { connect } from 'react-redux'
+import { connect, useDispatch, useSelector } from 'react-redux'
 import classNames from 'classnames'
 import ethereumMainnet from 'images/ethereum_mainnet.svg'
 import ethereumRopsten from 'images/ethereum_ropsten.svg'
 import fuseToken from 'images/fuse_token.svg'
-import { getAccount, getProviderInfo } from 'selectors/accounts'
+import { getAccount, getProviderInfo, getAccountAddress } from 'selectors/accounts'
 import { formatWei } from 'utils/format'
 import { getForeignNetwork, getCurrentNetworkType } from 'selectors/network'
 import { loadModal } from 'actions/ui'
 import { changeNetwork } from 'actions/network'
+import { fundEth } from 'actions/user'
 import { SWITCH_NETWORK } from 'constants/uiConstants'
 
+const Fund = ({ value, network, hasEth }) => {
+  const dispatch = useDispatch()
+  const accountAddress = useSelector(state => getAccountAddress(state))
+  return value === 'ropsten' && value === network && !hasEth && (
+    <button
+      onClick={(e) => {
+        e.preventDefault()
+        dispatch(fundEth(accountAddress))
+      }}
+      className='fund'
+      style={{ display: value === network && !hasEth ? 'block' : 'none' }}>
+      <span>
+        Click here to get 0.05 {value} eth
```

Comment:
Can you change to "Click here to get 0.05 {value} Ropsten ETH"

Reply:
The value in ropsten :)